packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -340,6 +341,38 @@ If a message attachment fails, an error message should share the reason for fail
 ![Error alert for failed attachment.](./img/attachment-error.svg)
 </div>
 
+### Downloading chat transcripts
+
+You can enable users to download chat transcripts, for their personal records or to share with others. When users choose to download a transcript, you can choose how you want to configure the behavior in your ChatBot. 
+
+For guidance, refer to our download transcripts demo, which opens a Markdown file for a conversation within a new tab. 
+
+Choose the download action location that best works for your ChatBot:
+
+#### Download via conversation history drawer
+
+If your ChatBot uses a conversation history drawer, you can provide a download option in the [actions menu linked to a previous conversation](/patternfly-ai/chatbot/ui#drawer-with-conversation-actions).
+
+<div class="ws-docs-content-img">
+![Expanded menu for previous chat in the history window, which shows a download option.](./img/download-chat-history.svg)
```

Comment:
Our demo text is a little different - do we want to change one or the other so they are consistent? Happy to change on my side if needed.

<img width="367" height="153" alt="Screenshot 2025-09-09 at 9 12 08 AM" src="https://github.com/user-attachments/assets/ebf82e90-b5a7-4195-91fd-5e98f1eef705" />

Reply:
We have our own staging that is a little bit ahead of PF staging: https://pf-virtual-assistant.surge.sh/

Staging won't have these until we're doing release prep!